docs/src/_data/rules.json

```diff
@@ -1893,7 +1893,7 @@
     ],
     "deprecated": {
         "name": "Deprecated",
-        "description": "These rules have been deprecated in accordance with the <a href=\"/docs/use/rule-deprecation\">deprecation policy</a>, and replaced by newer rules:",
+        "description": "These rules have been deprecated in accordance with the <a href=\"{{ '/use/rule-deprecation' | url }}\">deprecation policy</a>, and replaced by newer rules:",
```

Comment:
This file is autogenerated during releases, so we should also update the template:

https://github.com/eslint/eslint/blob/main/conf/rule-type-list.json

Otherwise, next release will overwrite these changes. You can run `node Makefile.js generateRuleIndexPage` to verify.

Reply:
Thanks for the catch, fixed in https://github.com/eslint/eslint/pull/16837/commits/946173f909348caf95760a01d8d52011adffb1d7